recipe/meta.yaml

```diff
@@ -1,58 +1,60 @@
-{% set name = "PhiK" %}
-{% set version = "0.11.2" %}
+{% set name = "phik" %}
+{% set version = "0.12.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/KaveIO/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: e98e6078b8c81165c6ad565b933883bda8370a375936c478f2d733bcd67863a8
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/phik-{{ version }}.tar.gz
+  sha256: b0674eb829d23290433f718d2360122be804b170ccc807ad9d9a8d1c13985533
 
 build:
-  number: 1
-  noarch: python
```

Comment:
Skip `py<37`, https://github.com/KaveIO/PhiK/blob/932f5402a1e5cd0eec948193ea7679092e036243/setup.py#L101

Reply:
Is this required since we support python 3.7 onwards?

Reply:
It's only for clarification and sync with the upstream source

Reply:
not convinced but I added it